README.rst

```diff
@@ -62,7 +66,9 @@ Python to send mail::
             # template_suffix="email",
     )
 
-If you would like finer control on sending the email, you can use **get_templated_email**, which will return a django **EmailMessage** object, prepared using the **vanilla_django** backend::
+If you would like finer control on sending the email, you can use **get_templated_email**, which will return a django **EmailMessage** object, prepared using the **vanilla_django** backend:
```

Comment:
Yea, removed all other backends. We should probably clean up and remove all the backends logic at some point.